flashchips.c

```diff
@@ -1517,6 +1517,45 @@ const struct flashchip flashchips[] = {
 		.voltage	= {2700, 3600}, /* 2.3-3.6V & 2.7-3.6V models available */
 	},
 
+	{
+		.vendor		= "Atmel",
+		.name		= "AT25DF021",
```

Comment:
Add the 'A' at the end since there is already a AT25DF021 chip.

Reply:
thanks, fixed it